Sources_v3/StreamChatUI/ChatChannel/Attachments/ChatMessageImageGallery.swift

```diff
@@ -21,7 +21,7 @@ open class ChatMessageImageGallery<ExtraData: ExtraDataTypes>: View, UIConfigPro
 
     public private(set) lazy var moreImagesOverlay: UILabel = {
         let label = UILabel()
-        label.font = UIFont.preferredFont(forTextStyle: .largeTitle).bold
+        label.font = .streamTitle // TODO: Not sure about this, check design for more fontStyles?
```

Comment:
Just ask in #mobile-design in Slack, they usually reply very quickly

Reply:
thanks, this is fixed now :)